src/renderers/dom/client/ReactDOMIDOperations.js

```diff
@@ -91,14 +90,14 @@ var ReactDOMIDOperations = {
    * @param {string} name A property name to remove, see `DOMProperty`.
    * @internal
    */
-  deletePropertyByID: function(id, name, value) {
+  deletePropertyByID: function(id, name) {
```

Comment:
Actually, I wonder if that's true for a number of things in here now that it's not being injected.